bundles/integration/admin-layerselector/templates/adminLayerSettingsTemplate.html

```diff
@@ -375,6 +375,7 @@
         </div>
     </div>
     
+    <% if (layerType === "wms" || layerType === "wmts") { %>
```

Comment:
This should be moved to wms and wmts wmslayerSettingsTemplateFooter.html and wmtslayerSettingsTemplateFooter.html.